tabs-and-subtabs/src/components/Tab/Tab.scss

```diff
@@ -0,0 +1,36 @@
+.navigation-list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  a.active {
+    background-color: grey;
+  }
+  
+  a.inactive {
+    background-color: lightgrey;
+  }
+  
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+  }
+  
+  li {
+    float: left;
+  }
+  
+  li a {
+    display: block;
+    color: black;
+    text-align: center;
+    padding: 16px;
+    text-decoration: none;
+  }
+  
+  li a:hover {
+    background-color: orange;
+  }
```

Comment:
targeting elements through tag names can be counterintuitive if there are multiple tags inside the component. Use proper classes instead.